### PR TITLE
DEV-2649: Fix entities list "filter to frustum" eye button

### PR DESCRIPTION
**Pull request by ctrlaltdavid**
Case: https://highfidelity.atlassian.net/browse/DEV-2649

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Can someone from High Fidelity approve this PR?

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/22355/HighFidelity-Beta-PR16461-eeceec3.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/22355/HighFidelity-Quest-Beta-PR16461-eeceec3.apk)

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/22355/HighFidelity-Beta-PR16461-eeceec3.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/22355/HighFidelity-Beta-Interface-PR16461-eeceec3.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/22355/HighFidelity-Beta-PR16461-eeceec3.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/22355/HighFidelity-Beta-Interface-PR16461-eeceec3.dmg

File: scripts/system/create/entityList/entityList.js [hunk 1]
```diff
@@ -177,7 +177,7 @@ EntityListTool = function(shouldUseEditTabletApp) {
 
             var cameraPosition = Camera.position;
             PROFILE("getMultipleProperties", function () {
-                var multipleProperties = Entities.getMultipleEntityProperties(ids, ['name', 'type', 'locked',
+                var multipleProperties = Entities.getMultipleEntityProperties(ids, ['position', 'name', 'type', 'locked',
                     'visible', 'renderInfo', 'modelURL', 'materialURL', 'imageURL', 'script', 'certificateID',
                     'skybox.url', 'ambientLight.url']);
```
Comment (sabrina-shanman) [inline scripts/system/create/entityList/entityList.js]: This change makes sense. Not a blocker: skybox.url and ambientLight.url appear to be unused.

**Pull request merged**